### PR TITLE
Improvements to Phoenix.LiveView.JS docs

### PR DESCRIPTION
**Pull request by rhcarvalho**
Like in the other "toggles" in this file, explicitly mention that classes may be added or removed based on presence.

### CODE REVIEW

File: lib/phoenix_live_view/js.ex [hunk 1]
```diff
@@ -465,20 +465,20 @@ defmodule Phoenix.LiveView.JS do
   end
 
   @doc """
-  Toggles classes on an element.
+  Adds or removes element classes based on presence.
 
-    * `names` - The string of classes to add.
+    * `names` - A string with one or more class names to toggle.
```
Comment (SteffenDE) [inline lib/phoenix_live_view/js.ex:470]: Changing the `the`s here and below makes this somewhat inconsistent with the other functions in this module, e.g.  `toggle`, `add_class`, etc.

File: lib/phoenix_live_view/js.ex [hunk 2]
```diff
@@ -465,20 +465,20 @@ defmodule Phoenix.LiveView.JS do
...
+    * `:to` - The optional DOM selector to target.
       Defaults to the interacted element.
-    * `:transition` - The string of classes to apply before adding classes or
+    * `:transition` - A string of classes to apply before adding classes or
       a 3-tuple containing the transition class, the class to apply
       to start the transition, and the ending transition class, such as:
       `{"ease-out duration-300", "opacity-0", "opacity-100"}`
-    * `:time` - The time to apply the transition from `:transition`.
-      Defaults #{@default_transition_time}
+    * `:time` - The time in milliseconds to apply the transition from `:transition`.
```
Comment (SteffenDE) [inline lib/phoenix_live_view/js.ex:480]: if we add this here, I think we should mention it in the docs for the other functions as well

File: lib/phoenix_live_view/js.ex (hunk 1 above)
Comment (rhcarvalho) [reply lib/phoenix_live_view/js.ex:470]: That's true. I wanted to keep the changes contained but noticed the inconsistency. I'll go ahead and update the rest.

Comment (rhcarvalho) [reply lib/phoenix_live_view/js.ex:471]: Done

**Comment by rhcarvalho** [comment]
@SteffenDE thanks for the review. I've rebased against latest main (no conflicts and no changes to my original commit), and applied the suggestions revising the file/module as a whole as a new commit.

**Comment by SteffenDE** [comment]
lgtm. Let's maybe get some feedback from @chrismccord as well as he wrote most of the docs for the JS module.

**Comment by chrismccord** [comment]
❤️❤️❤️🐥🔥

**Pull request merged**